docs/import/running-the-import.md

```diff
@@ -0,0 +1,76 @@
+# How to run the import pipeline
+
+Configure settings:
+```bash
+# Repository and branch used for the run. Useful for testing on different branches.
+export REPOSITORY=https://github.com/EBIvariation/metadata-ws
+export BRANCH=master
+
+# Path where Maven is installed (or will be installed)
+# See also instructions below
+export MAVEN_INSTALL_PATH=
+export PATH=$MAVEN_INSTALL_PATH/bin:$PATH
+
+# Oracle credentials to download the database drivers
+# See Confluence → Variation home → IT infrastructure → External services credentials → Oracle
+export ORACLE_USERNAME=
+export ORACLE_PASSWORD=
+
+# Entrez API key to do batch queries
+# See Confluence → Variation home → IT infrastructure → External services credentials → NCBI Entrez API key 
+export ENTREZ_API_KEY=
+
+# Source database credentials (ENA)
+# See Confluence → Variation home → IT infrastructure → Database Connections → ENA database credentials
+export ENA_DATASOURCE_URL=
+export ENA_DATASOURCE_USERNAME=
+export ENA_DATASOURCE_PASSWORD=
+
+# Target database (Postgres), which will be used to store the import results
+# See Confluence → Variation home → IT infrastructure → Database Connections → AMP metadata staging database
+export METADATA_DATASOURCE_URL=
+export METADATA_DATASOURCE_USERNAME=
+export METADATA_DATASOURCE_PASSWORD=
+```
+
+If you don't have Maven, install it. Note that the version used on the cluster (3.0.5) has a bug and cannot read `settings.xml` correctly.
+```bash
+mkdir -p $MAVEN_INSTALL_PATH
+cd $MAVEN_INSTALL_PATH
+wget -q http://us.mirrors.quenda.co/apache/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz
+tar --extract --verbose --file=apache-maven-3.6.3-bin.tar.gz
+rm apache-maven-3.6.3-bin.tar.gz
+mv apache-maven-3.6.3/* .
+rmdir apache-maven-3.6.3
+```
+
+Clone the repository and build the package:
+```bash
+git clone $REPOSITORY
+cd metadata-ws
+git checkout $BRANCH
+sed -ie "s|<username></username>|<username>${ORACLE_USERNAME}</username>|" settings.xml
+sed -ie "s|<password></password>|<password>${ORACLE_PASSWORD}</password>|" settings.xml
+mvn -s settings.xml clean install -DskipTests=true
+```
+
+Write the list of studies to import to `studies-to-import.txt`.
+
+Run the import:
+```bash
+time bsub -M 20G -I java -Xmx18g \
```

Comment:
Also add the -Rusage switch...

Reply:
Thanks, fixed in https://github.com/EBIvariation/metadata-ws/pull/86/commits/608ed7c7bf51ffab28ecd9637833f8cf5bf94477